### ISSUE TITLE
Sonoff POW R2 - Mqtt error

### ISSUE THREAD

**Issue by arek156** [xoseperez/espurna#939]
Hello
I have a problem with sonoff pow r2, after uploading espurna everything works until the mqtt client starts. After turning on the mqtt, i can see a significant delay in switching, additionally at various intervals, the sonoff is switched off and finally the wifi network is lost. I use other Sonoff S20 devices and espurna software and everything works, but not pow r2, I have the latest of espurna soft 1.12.6
Do you have any idea what can cause mqtt error with this switch?

Regards
@rek

**Comment by xoseperez** [comment]
Is it a local MQTT broker? What configuration are you using? It could be helpful to see the debug log while sending MQTT commands to the device.

**Comment by arek156** [comment]
local mqtt broker, installed on raspberry pi, port 1883

**Comment by xoseperez** [comment]
Can you share a debug log showing the issue?

**Comment by arek156** [comment]
[log_sonoff_pow2.txt](https://github.com/xoseperez/espurna/files/2102962/log_sonoff_pow2.txt)

**Comment by arek156** [comment]
Is this log correct?

**Comment by arek156** [comment]
Hi, I've uploaded the software again today. This time mqtt works, however, after switching the sonoff to ON, it automatically switches to OFF at different time intervals.

**Comment by xoseperez** [comment]
Questions/issues:
* I can only see a relay change request from the web interface
* I don't see any relay OFF request in the log
* Could it be your domoticz turning the relay off?
* Looks like your MQTT root topic is not set.
* Sending a temperature value every 2 seconds is a little overkill, isn't it?

**Comment by xoseperez** [comment]
You can set the root topic in the MQTT tab of the web interface. You don't have pulses enabled, do you?

**Comment by arek156** [comment]
Hi, I completed the data in mqtt root topic, however it does not help. I noticed that the problem only occurs if I enable mqtt support. I am using sonoff bridge and sonoff s20 on your software and everything works, so I put a problem on the sonoff pow r2 side. I also installed espurna on sonoof basic and it also works perfectly, I will try to clear the memory and upload the soft one again, maybe it will help.

P.S I don't have pulse enabled
